### DISCUSSION TITLE
InfluxDB v2 API needs read AND write to work

### DISCUSSION THREAD

**Discussion by MijnSpam** [johanmeijer/grott#543] [General]
Hi, 
I created a API token with only write rights and then grott service fails to start. When I made a second API token with both read and write rights the service starts correctly. This isn't mentioned in the Influx readme.
The error message bucket xxx doesn't exits doesn't really gives a full representation of the possible error.

**Comment by johanmeijer** [comment]
Grott is testing at the beginning if the bucket exist. I can understand it need read writes for that. I will see if I can change the error message.
